scripts/gitcollab.sh

```diff
@@ -56,18 +56,20 @@ function test-integration() {
     source gitcollab_pyenv/bin/activate && pytest ./integration_tests/
 }
 
-function fetch_valid_containers() {
+function fetch_valid_containers() {\
```

Comment:
should slash be here?

Reply:
prob not

Reply:
remove 
![image](https://user-images.githubusercontent.com/29551974/218282399-cba0d71a-222b-43d8-8148-ff178c878cda.png)